src/tribler-core/tribler_core/components/tag/community/tag_payload.py

```diff
@@ -0,0 +1,36 @@
+from types import SimpleNamespace
+
+from pydantic import BaseModel
+
+from ipv8.messaging.lazy_payload import VariablePayload, vp_compile
+
+
+class TagModel(BaseModel):
+    infohash: str
+    tag: str
+    creator_public_key: str
+    creator_sign: str
```

Comment:
**Issue:** There currently is no primitive to remove tags. Earlier, we had a brief discussion on whether to use different message types to do so, or to add an additional flag in each payload (e.g., `removed` that is set to `1` if the tag is removed). I would opt for adding this flag in a payload so that we have a single message type that is being gossiped around.

Reply:
Agree. Will add this (or something like this) in near future.

The PR is a draft. It is not ready yet.

I've published it only for one reason: to give a rough view of how it will look like.

Reply:
A primitive to remove tags has been added.